builder/dockerfile/instructions/commands.go

```diff
@@ -119,6 +119,21 @@ type LabelCommand struct {
 	Labels KeyValuePairs // kvp slice instead of map to preserve ordering
 }
 
+// NewLableCommand creates a new 'LABLE' command
+func NewLableCommand(k string, v string) *LabelCommand {
```

Comment:
You have "Lable" rather than "Label" three times in these two lines.

Reply:
Hello @ijc , thanks for spotting this  😛 will correct in the 2nd version...